content/en/docs/reference/components/docker-registry.md

```diff
@@ -79,7 +79,7 @@ kubectl create secret generic jenkins-docker-cfg --from-file=./config.json
 
 ## Using Docker Hub
 
-If you want to publish images to docker hub then you need to modify your `config.json` as described above to something like:
+If you want to publish images to docker hub then you need to modify your `config.json` in your home directory as described above to something like:
```

Comment:
in your home directory? That seems wrong. I guess you are referring to "~/.docker/config.json", right?

Reply:
yes you are correct

Reply:
do you mind updating the PR?

Reply:
done pls check, thx for your remind